tests/gke_zero_node_pools.tf

```diff
@@ -17,6 +17,16 @@ module "gke_zero_node_pool" {
       location       = module.gke_zero.current_config["region"]
       node_locations = ["europe-west1-c", "europe-west1-d"]
 
+      guest_accelerator = {
+        type               = "nvidia-testla-k80"
+        count              = 1
+        gpu_partition_size = "1g.5gb"
+        gpu_sharing_config = {
+          gpu_sharing_strategy       = "TIME_SHARING"
+          max_shared_clients_per_gpu = 2
+        }
+      }
+
```

Comment:
Sorry, but this would mean I have to pay for GPUs for all the many clusters I create during testing. I do not want to spend my money on that. Please remove this.